src/components/hero/hero.scss

```diff
@@ -0,0 +1,127 @@
+/* HeroMlhBadge */
```

Comment:
Let's split this file up into a series of separate sass files for each component

Reply:
Good idea, I was thinking of renaming the folder to 'landing-page' and having 'hero' and 'navbar' as sub-folders within it. Let me know what you think.

Reply:
Good idea